src/cli/backup/backup.go

```diff
@@ -4,6 +4,20 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// ==============================================
+// Folder Object flags
+// These options are flags for indicating
+// that a time-based filter should be used for
+// within returning objects for details.
+// Used by: OneDrive, SharePoint
+// ================================================
+var (
+	fileCreatedAfter   string
+	fileCreatedBefore  string
+	fileModifiedAfter  string
+	fileModifiedBefore string
+)
+
```

Comment:
Nit: Can we move these to a file like `flags.go` in the same package?  It'll be a more extensible location than this file.  In fact, I'd say that /utils/flags.go is an even better location (exported, so that flag declarations can reference them directly), since these attributes are common across both `backup details` and `restore` commands.

Reply:
Agreed

Reply:
Resolved within [d7988e7](https://github.com/alcionai/corso/pull/2679/commits/d7988e7b36145e689e75375b74cb1009c42d049d)